salt/metalk8s/metalk8s-ui/files/metalk8s-ui_deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: ui
+    kubernetes.io/name: "MetalK8sUI"
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      k8s-app: ui
+  template:
+    metadata:
+      labels:
+        k8s-app: ui
+    spec:
+      tolerations:
+        - key: "node-role.kubernetes.io/master"
+          operator: "Exists"
+          effect: "NoSchedule"
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      containers:
+      - name: metalk8s-ui
+        image: localhost:5000/metalk8s-2.0/metalk8s-ui:0.2
```

Comment:
@nootal This will need to be adapted somehow? Not really a blocker for now, I guess.